src/App.tsx

```diff
@@ -1,61 +1,130 @@
+import { useState } from 'react';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+const findUser = (userId: number) => (
```

Comment:
Use more descriptive names.
Add verb as first word in name if you creating function. Verb must describe which action that function will do
```suggestion
const getUserById = (userId: number) => (
```